dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::UserSchoolInfosController < ApplicationController
   before_action :authenticate_user!
-  before_action :load_user_school_info
-  load_and_authorize_resource
+  load_and_authorize_resource only: :update_last_confirmation_date
+  skip_before_action :verify_authenticity_token
```

Comment:
I think we don't want to skip token verification; not 100% sure about this though.